### DISCUSSION TITLE
Normalization factor

### DISCUSSION THREAD

**Discussion by JSuryatenggara** [fndemello/CUT-RUN_greenlist#1] [Q&A] [answered]
Thank you so much for improving the normalization of CUT&RUN signals. I have been working on CUT&RUN pipeline for quite a while and it would be an understatement that a consistent and robust normalization method across varying datasets is highly needed. 

I have a couple of questions. First, if I want to generate normalized bedgraph and bigwig files from each bam file, how is the best way to do it? Will using the numbers in "glist_sizeFactors.tsv" resulting from "In [38]" command line box as the scaling factor be the correct course of action?

Second, I am guessing that for the application of dataset normalization, since you have already provided the green list bed files, users do not need to calculate entropy nor call peaks anymore. Am I correct to assume this?

Thank you very much!

**Comment by fndemello** [comment, marked as answer]
Thank you for your kind comments! I'm glad I could help :)

Regarding your questions, yes, the output of "glist_sizeFactors.tsv" are the final scaling factors to be used. The first column is directly proportional to the counts (which is how DESeq2 prefer it, for example) and the second column is inversely proportional. 

For bedgraphs and bigWigs, [deeptools's bamCoverage](https://deeptools.readthedocs.io/en/develop/content/tools/bamCoverage.html) works really well, you just supply the second column (inversely proportional) to bamCoverage's "--scaleFactor" option.

Secondly yes, your assumption is spot on. The whole workflow of calculating entropy and defining greenlist regions is only meant for user applying the method to different organisms. If you're just working with human or mouse, you can just use the provided bed files.

**Reply by JSuryatenggara** [reply]
Understood. Thank you so much for the detailed response!

**Comment by JSuryatenggara** [comment]
Hi, it's me again. I have been using the greenlist method for normalization for a while now and I liked how stable it is at wildly varying sequencing depths. I am currently optimizing a pipeline for CUT&RUN which handles multiple genomes (hg19, mm9, dm6, sacCer3, etc), and therefore, I would also need a greenlist for each of those. Do you have some kind of tutorial (akin to what's in CUTandRUNAnalysis.ipynb) on how to generate the greenlists. Thanks a lot!
